src/pods/canvas/canvas.business.ts

```diff
@@ -42,3 +43,26 @@ export const findField = (fields: FieldVm[], id: GUID): FieldVm | undefined => {
   }
   return undefined;
 };
+
+//TODO: 300 that's the width of the table and we will have to treat this in a separate case
+export const calculateRelationXCoordinateOrigin = (
+  tableOrigin: TableVm,
+  tableDestination: TableVm
+): number =>
+  tableOrigin.x < tableDestination.x ? tableOrigin.x + 300 : tableOrigin.x;
```

Comment:
Let's move this 300 harcoded value into a const, by doing that it can be easier to replace in the future

You can move it to a file like canvas.const.ts

And add a const like

```ts
const DEFAULT_TABLE_WITH_VALUE = 300;
```

Then we can replace all the harcoded _300_ with that const

Reply:
It's fixed.